relay-general/src/protocol/span.rs

```diff
@@ -35,6 +35,12 @@ pub struct Span {
     /// The status of a span
     pub status: Annotated<SpanStatus>,
 
+    /// Arbitrary tags on a span, like on the top-level event. Will maybe be indexed in the future.
+    pub tags: Annotated<Object<LenientString>>,
```

Comment:
Probably want to allow more flat types like bools and ints here.

Reply:
Yeah I am convinced we need at least int here because of http status

Reply:
Tag values are always strings though?

Reply:
You sure? I thought we stored http status as int

Reply:
Ah you're right. I was thinking about tags from the product side.

Reply:
turns out almost everything is string, so let's stick with string for now